pkg/controller/storagecluster/storagecluster.go

```diff
@@ -306,6 +308,23 @@ func (c *Controller) validateCustomAnnotations(current *corev1.StorageCluster) e
 
 // RegisterCRD registers and validates CRDs
 func (c *Controller) RegisterCRD() error {
+	k8sVersion, err := k8s.GetVersion()
+	if err != nil {
+		return fmt.Errorf("error parsing Kubernetes version '%s'. %v", k8sVersion, err)
+	}
+
+	k8s1_16, err := version.NewVersion("1.16")
```

Comment:
I am thinking whether we should use 1.22 or 1.16 here, if we use 1.22 then we only need to test new operator with 1.22, if we use 1.16 we may need to test the new operator with 1.16-1.21 to be safe. I am ok with using 1.16, just want to see if anyone foresee any issue. we need to test it prior to release.

Reply:
Testing effort is a very good point. I assumed we have a kubernetes matrix job where we test multiple k8s versions and also k8s upgrades. Let me confirm with the QA team. 
Honestly, the only reason I added 1.16 here is to get the 'kubectl explain storagecluster|storagenode' working (https://portworx.atlassian.net/browse/OPERATOR-210). It did not work before because our schema was not structural.

If we are not covering the below scenarios in the automation, then we can do this only for 1.22 and above. Wdyt?